LORIS_indepth.html

```diff
@@ -138,141 +67,43 @@ <h4 align="center"><font color="f27c30">Heterogenous data acquisition</font></h4
-			</td>
-			<td width="25%" style="vertical-align:middle">
-				<img src="images/public_dataset.png" style="border:0">
-			</td>
-			</tr></table> -->
-          </section>
-
-<!-- WHAT DOES LORIS DO? -->
-<section>
-            <h2>What does LORIS do?</h2>
-            <img src="images/LORIS_functionality.png">
-          </section>
-
-<!-- PERFERRED SLIDE WHAT DOES LORIS DO -->	  
-	  <section>
+            <img src="images/loris_versatility.png" align="center" width="90%" style="border:0">
+            <p><small>A flexible, customizable, and <font color="f27c30">open source</font> neuroimaging <br> database suitable for multiple types of projects.</small></p>
+            </section>          
+              </section>
+              
+              <!-- What does LORIS do? Column 2 -->
+              
+        <section>
+               <section>
 	    <h2>What does LORIS do?</h2>
-	    <img src="images/loris_purpose.jpg" style="border:0" width="80%"> 
+	    <img src="images/LORIS_hexagons.png"> 
 	  </section>
-	 
-	  
-	 <!-- LORIS Key features -->
-                      <section data-background="images/loris_features.jpg">
-         <!--         <h2><font color="0d4983">Key Features</font></h2>-->
-                      </section>
+            
+            <section>
+                <h3>LORIS Around the World!</h3>
+            <img src="images/LORIS_map.png">
+            </section>
+            
+               <section>
+            <h2>What does LORIS <br> actually do?</h2>
 
- 			
-		<section>
-                        <h2>Extensible and modular design</h2>
-                        <table class="reveal">
-                            <tr>
-                              <td width="60%" style="vertical-align: middle">
-                                <img src="images/Dashboard.png">
-                               	<img src="images/Imaging_scans.png">
-                              </td>
+	    <table class="reveal">
+	      <tr>
+	      	<td width="60%" style="vertical-align:middle">
 
-                              <td width="40%" style="vertical-align: middle">
-                        	<img src="images/BrainBrowser_VolumeViewer.png">
-                             	<img src="images/Statistics_behavioural.png">
-                              </td>
-                            </tr>
-                           </table>
-                      </section>
-	  <section data-background="images/LORIS_global_2.jpg" data-background-size="99%">
-	    <h1 style="margin:350px 0px 0px 0px;"><font color ="0d4983">LORIS Globally</font></h1>
+		  <h4 align="center"><font color="f27c30">DATA ENTRY<br>DATA MANAGEMENT<br>DATA VISUALISATION<br>DATA QUERYING<br></font></h4>
+		  <span class="fragment"><h3 align="center"><a href="https://demo.loris.ca" target="_blank">Try the LORIS Demo</a></h3></span>
+		</td>
+		<td width="40%" style="vertical-align:middle">
+		  <img src="images/loris_arch.png" style="padding:0; margin:0;">
```

Comment:
image is too small... is it because the text is now given enough space to avoid wrap?  
can the left/right buffer on the slide be diminished instead?   
it seems either the text or the image generally has to suffer with the wide margins we have currently

Reply:
tried to change this! played around with the width of each column and the image is bigger now, although the text doesn't look as good...